templates/taxi/car_list.html

```diff
@@ -1,16 +1,43 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a style="float: right" href="{% url 'taxi:car-create' %}">+</a></h1>
+  
   {% if car_list %}
-    <ul>
+    <table class="table">
+        <tr>
+          <th>ID</th>
+          <th>Model</th>
+          <th>Manufacturer</th>
+          <th>Drivers</th>
+          <th>Update</th>
+          <th>Delete</th>
+        </tr>
+
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
+        <tr>
+          <td>
+              {{ car.id }}
+          </td>
+          <td>
+              {{ car.model }}
+          </td>
+          <td>
+              {{ car.manufacturer }}
+          </td>
+          <td>
+              {{ car.drivers }}
```

Comment:
The `{{ car.drivers }}` might not display the drivers correctly if `drivers` is a related field (e.g., a ManyToManyField). Consider iterating over `car.drivers.all` to list each driver associated with the car.